src/engine/rtr_inputs/rtr_clangformat.json

```diff
@@ -0,0 +1,11 @@
+{
+    "steps": [
+        {
+            "description": "Coding style",
+            "parameters": [
+                "--test format",
+                "--exclude build"
+            ]
+        }
+    ]
+}
```

Comment:
That option exists but currently isn't presented in any example. 
I prefer, as a RTR user, to have it by separate because I find it "invasive" to change all the source code without previous confirmation.

I'll add some extra JSON to use that flag.